src/components/BaseIcon/index.tsx

```diff
@@ -27,6 +28,7 @@ const icons = {
   ["notification-off"]: NotificationOff,
   ["paint-brush"]: PaintBrush,
   ["user-group"]: UserGroup,
+  ["upload-cloud"]: UploadCloud,
```

Comment:
N achei nenhum icon lá `upload-cloud`, será q tu n confundiu?

```suggestion
  ["upload-cloud-1"]: UploadCloud1,
```

```suggestion
  ["upload-cloud-2"]: UploadCloud2,
```